bees/cronbee/cronbeefactory.go

```diff
@@ -91,12 +91,12 @@ func (factory *CronBeeFactory) Options() []bees.BeeOptionDescriptor {
 		},
 		{
 			Name:        "day_of_month",
-			Description: "01-31 for a specific time; * for ignore",
+			Description: "01-31 for a specific day of month; * for ignore",
```

Comment:
I would call this "day of the month" (note the extra 'the') as you did in timebee, too.